### PR TITLE
Add more validations in mask operations

### PR DESCRIPTION
**Pull request by Avogar**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Review by CurtizJ — APPROVED** [review]

**Review by CurtizJ — COMMENTED** [review]
Ah, too premature approve, tests are failing

File: src/Columns/MaskOperations.cpp [hunk 1]
```diff
@@ -120,6 +123,10 @@ size_t extractMaskNumericImpl(
 
         mask[i] = value;
     }
+
+    if (i != mask_size || data_index != data_size)
```
Comment (CurtizJ) [inline src/Columns/MaskOperations.cpp:127]: Probably need another check where `column_is_short` is false.

Comment (Avogar) [reply src/Columns/MaskOperations.cpp]: Yep, forgot to check `column_is_short`. Updated

**Comment by Avogar** [comment]
@Mergifyio update

**Pull request merged**